docs/guides/update-guide/introduction.md

```diff
@@ -20,6 +20,15 @@ Versions prior to Camunda Platform 8 are listed below and identified as Camunda
 
 There is a dedicated update guide for each version:
 
+### [Camunda Platform 8.2 to Camunda Platform 8.3](../820-to-830)
+
+Update from 8.2.x to 8.3.0
+
+<!-- ADD LINK WHEN AVAILABLE
```

Comment:
Should we place a hold label until this is released, or create another issue so we don't forget?

Reply:
I create this [issue](https://github.com/camunda/camunda-platform-docs/issues/2377)